.gitignore

```
@@ -9,4 +9,7 @@
 *.ai
 *.gif
 */_targets/
-*/scratch.R
\ No newline at end of file
+*/scratch.R
+04_flora_cnell/*.tiff
```

This is not actually ignoring the tiffs because they are within `04_flora_cnell/out/`. Slight adjustment here, plus I suggest that the PNGs created are ignored, too.
```suggestion
04_flora_cnell/out/*.tiff
04_flora_cnell/out/*.png
```